build.gradle

```diff
@@ -42,6 +46,13 @@ subprojects {
 }
 
 project(':accurest-core') {
+	dependencies {
+		compile 'commons-io:commons-io:2.4'
+		compile 'org.codehaus.plexus:plexus-utils:3.0.21'
+		testCompile 'cglib:cglib-nodep:2.2'
```

Comment:
Commons-io to wildcard matching files, plexus-utils for ant-style directory exclusions, and cglib/objenesis for spock mocking.